packages/commonjs/README.md

```diff
@@ -127,6 +127,32 @@ Default: `[]`
 
 Sometimes you have to leave require statements unconverted. Pass an array containing the IDs or an `id => boolean` function. Only use this option if you know what you're doing!
 
+### `dynamicRequireTargets`
+
+Type: `String|Array[String]`<br>
+Default: `[]`
+
+Some modules have dynamic `require` calls in them, or even circular dependencies (which are not handled well by EJS).  
```

Comment:
```suggestion
Some modules contain dynamic `require` calls, or require modules that contain circular dependencies, which are not handled well by [EJS](link to EJS info).  
```

What is EJS? Uncommon acronyms without links to info about them are generally unfriendly to new users. Please add a link to info about EJS.

Reply:
Wrong teminology.